rosdep/python.yaml

```diff
@@ -6107,6 +6107,14 @@ python3-sexpdata:
     xenial:
       pip:
         packages: [sexpdata]
+python3-shapely:
+  arch: [python-shapely]
+  debian: [python3-shapely]
```

Comment:
The `fedora` package database seems to be not working at the moment.  I can definitely do `dnf install python3-shapely` on my Fedora box, so I'm guessing they are just having some backend issues at the moment.  Not your fault, but please do add the key for Fedora as well.